go1.14/amd64/rootfs/compilers.yaml

```diff
@@ -0,0 +1,25 @@
+---
+
+windows:
+  386:
```

Comment:
The 386 compilers are no longer in this image IIUC so the `386` entries can be dropped. I think of this file as kind of a manifest so it's helpful to have it match up with what GOOS/GOARCH combos we expect the image to be used for.

Reply:
yep, I saw it, I reviewing all these `compilers.yaml` files